hwdb.d/60-keyboard.hwdb

```diff
@@ -1276,6 +1276,16 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnMicro-Star*:pn*A10SC*:*
  KEYBOARD_KEY_f1=f20
  KEYBOARD_KEY_f2=f21
 
+# MSI Modern series
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnMicro-StarInternational*:pnModern*:*
+ KEYBOARD_KEY_f1=f20                                    # Fn+F5 micmute
+ KEYBOARD_KEY_76=f21                                    # Fn+F4 touchpad, becomes meta+ctrl+toggle
+ KEYBOARD_KEY_91=prog1                                  # Fn+F7 Creation Center, sometime F7
+ KEYBOARD_KEY_f2=prog2                                  # Fn+F12 screen rotation
+ KEYBOARD_KEY_97=unknown                                # lid close
+ KEYBOARD_KEY_98=unknown                                # lid open
+                                                        #Fn+PrntScr sends meta+shif+s
```

Comment:
comment is weird indented and truncated?

Reply:
I just want to leave info comment about the keyboard behavior, and yep, it will be better to meta+shif**t**+s. My new keyboard are still inappropriately for me)